src/main/java/org/jenkinsci/maven/plugins/hpi/HpiMojo.java

```diff
@@ -109,18 +121,11 @@ private void performPackaging()
         generateManifest(archive, manifestFile);
         Manifest manifest = loadManifest(manifestFile);
 
-        getLog().info("Checking for attached .jar artifact "
+        getLog().info("Checking for attached .jar `artifact "
```

Comment:
Please remove this inadvertent change
```suggestion
        getLog().info("Checking for attached .jar artifact "
```